app/models/work_experience.rb

```diff
@@ -1,4 +1,13 @@
 class WorkExperience < ApplicationRecord
-	validates :employer, :occupation, :start_date, :end_date, presence: true
+	validates :employer, :occupation, :start_date, presence: true
+  validates :end_date, presence: true, :unless => :current_job?
 	belongs_to :application
+  
+  def current_job(answer)
+    @answer = answer == 'on' ? true : false
+  end
+
+  def current_job?
+    @answer
```

Comment:
I don't think we should be checking an instance variable that may or may not be set. Anything wrong with adding an attribute to the WorkExperience model called `current_job` that is a boolean field? I think that's a more reliable approach here.